server/cluster.go

```diff
@@ -325,10 +325,12 @@ func (s *Server) bootstrapCluster(clusterID uint64, req *pdpb.BootstrapRequest)
 	ops = append(ops, clientv3.OpPut(regionSearchPath, string(regionValue)))
 
 	bootstrapCmp := clientv3.Compare(clientv3.CreatedRevision(clusterRootPath), "=", 0)
-	resp, err := s.client.Txn(context.TODO()).
+	ctx, cancel := context.WithTimeout(context.Background(), requestTimeout)
+	resp, err := s.client.Txn(ctx).
```

Comment:
The transaction may success or fail when timeout. We need to handle it carefully.